apps/balancer-tools/src/lib/gql/server.ts

```diff
@@ -91,3 +92,9 @@ export const blocks = {
   gql: (chainId: string) =>
     wrapWithCache(blocksSdks[networkFor(chainId)](blocks.client(chainId))),
 };
+
+export const rewards = {
+  client: clientFor(Subgraph.BalancerRewards),
+  gql: (chainId: string) =>
+    rewardsSdks[networkFor(chainId)](rewards.client(chainId)),
```

Comment:
@ribeirojose this should be `wrapWithCache`, right?

Reply:
not sure -- but we need to be careful here because if your query is doesn't have a time parameter we could be caching things we don't want to cache here